src/main/java/co/rsk/federate/signing/hsm/message/AdvanceBlockchainMessage.java

```diff
@@ -23,4 +27,22 @@ private String parseBlockHeader(BlockHeader blockHeader) {
     public List<String> getBlockHeaders() {
         return blockHeaders;
     }
+
+    public List<String[]> getBrothersList() {
+        return brothersList;
+    }
+
+    private List<String[]> parseBrothersBlockHeaders(List<List<BlockHeader>> brothersList) {
+        List<String[]> result = new ArrayList<>();
```

Comment:
```suggestion
        List<String[]> parsedBrothers = new ArrayList<>();
```
Let's use meaningful variable names

Reply:
Updated